sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/ReadSchemaTest.scala

```diff
@@ -108,6 +108,40 @@ trait AddColumnTest extends ReadSchemaTest {
   }
 }
 
+trait AddColumnIntoTheMiddleTest extends ReadSchemaTest {
```

Comment:
Can we also add few lines of comment for this trait as `AddColumnTest`?

Reply:
Oops. I missed that. Thanks.